Cards/MadGraph5_aMCatNLO/DY/DYto2L-2Jets_MLL-10to50_amcatnloFXFX-pythia8/DYto2L-2Jets_MLL-10to50_amcatnloFXFX-pythia8.json~

```diff
@@ -0,0 +1,27 @@
+{
```

Comment:
Is this file needed? It has ~ in the back

Reply:
my mistake